db/migrate/20220720124734_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string
+  end
```

Comment:
- [ ] I am facing this error when I run the migration 👇🏻 Please fix this:

![migrat](https://user-images.githubusercontent.com/79058364/180088079-57b3d057-3057-4375-ac2a-70e93a926dd5.png)

Reply:
Hie @BenMukebo you have to run rails db:reset db:create db:migrate db:seed

Reply:
Hello @tafaramafemba, @kemigabocatherine Please go and ask for a re-revious, the second code reviewer will look at and if everything run as expected the project will be approved !!